LESS/index.less

```diff
@@ -9,9 +9,44 @@
 // Import Article into index.less
 @import '../Article/Article.less';
 
-html, body {
+html,
+body {
   margin: 0;
   padding: 0;
   font-family: 'Roboto';
   background-color: #fffef7;
+}
+
+// .article h2 {
+//   animation: article h2 0.7s cubic-bezier(0.215, 0.610, 0.355, 1.000) both;
+// }
+
+// @keyframes article h2{
+//   0% {
+//     letter-spacing: -0.5em;
+//     opacity: 0;
+//   }
+
+//   40% {
+//     opacity: 0.6;
+//   }
+
+//   100% {
+//     opacity: 1;
+//   }
+// }
+
+
+.articles .article p:nith-child(n+2) {
```

Comment:
Your animation's not working because `nth` was misspelled. 😅 I recommend downloading [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker), it helped me figure this bug out with that extension. 😉